lib/vsc/install/ci.py

```diff
@@ -62,19 +65,68 @@
 PY3_ONLY = 'py3_only'
 PY3_TESTS_MUST_PASS = 'py3_tests_must_pass'
 RUN_SHELLCHECK = 'run_shellcheck'
+ENABLE_GITHUB_ACTIONS = 'enable_github_actions'
 
 logging.basicConfig(format="%(message)s", level=logging.INFO)
 
 
 def write_file(path, txt):
     """Write specified contents to specified path."""
+
+    if not os.path.exists(os.path.dirname(path)):
+        os.makedirs(os.path.dirname(path))
     try:
         with open(path, 'w') as handle:
             handle.write(txt)
         logging.info("Wrote %s", path)
     except (IOError, OSError) as err:
         raise IOError("Failed to write %s: %s" % (path, err))
 
+def gen_github_action():
+    """
+    Generate tox.ini configuration file for github actions.
+    """
+    logging.info('[%s]', GITHUB_ACTIONS)
+    vsc_ci_cfg = parse_vsc_ci_cfg()
+
+    name_url = "https://github.com/hpcugent/vsc-install"
+
+    if vsc_ci_cfg[ENABLE_GITHUB_ACTIONS]:
+        header = [
+            "%s: configuration file for github actions worflow" % GITHUB_ACTIONS,
+            "This file was automatically generated using 'python -m vsc.install.ci'",
+            "DO NOT EDIT MANUALLY",
+        ]
+
+        txt = ['# ' + l for l in header]
+        yaml_content = {
+            'name': 'run python tests',
+            'on': ['push', 'pull_request'],
+            'jobs': {
+                'python_unittests': {
+                    'runs-on': 'ubuntu-20.04',
```

Comment:
Maybe directly run it inside an rhel 8 container? That's our production environment anyway.

Reply:
according to https://github.com/actions/runner-images there is no rhel 8 container.

Reply:
Not a runner image, a container. Just add a line like: 
```
    runs-on: ubuntu-20.04
    container: rockylinux:8
    steps:
```
Or whatever your favorite rhel flavour is.